DataBaseTutorial/ProductService.cs

```diff
@@ -58,7 +65,12 @@ public void ValidateAndAddNewProduct(string productName, string productWeight, b
         /// <returns></returns>
         public List<Product> GetProductList()
         {
-            return _myProducts;
+            using (DatabaseContext dbContext = new DatabaseContext())
+            {
+                dbContext.Product.Add(new Product());
```

Comment:
Why do you add new empty product when getting product list. Consider removing this part